README.md

```diff
@@ -145,6 +145,7 @@ zone.  This role has a trust relationship with the users account.
 | [aws_route53_record.staging_ready_set_cyber_prod_AAAA](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/route53_record) | resource |
 | [aws_route53_record.staging_ready_set_cyber_staging_A](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/route53_record) | resource |
 | [aws_route53_record.vincent_dev_CNAME](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/route53_record) | resource |
+| [aws_route53_record.vincent_dev_cert_verify_CNAME](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/route53_record) | resource |
```

Comment:
You should properly alphabetize the resources by name:
```suggestion
| [aws_route53_record.vincent_dev_cert_verify_CNAME](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/route53_record) | resource |
| [aws_route53_record.vincent_dev_CNAME](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/route53_record) | resource |
```

Reply:
See commit 82ced83.

Reply:
Fixed. Apologies as I was set up to work with this repo on a system that does not have terraform installed. I'll be changing my workflow here to use a system that does have terraform so that I can generate this documentation automatically for future requests.